src/microbe_stage/Microbe.cs

```diff
@@ -1068,8 +1087,19 @@ public override void _Process(float delta)
         {
             // Redo the cell membrane.
             SendOrganellePositionsToMembrane();
+
+            if (IsForPreviewOnly)
+            {
+                // Update once for the positioning of external organelles
```

Comment:
Isn't this comment wrong? This is inside _Process which gets called continuously.

Reply:
But this is only called when `membraneOrganellePositionsAreDirty` is true so it's not called everytime.

Reply:
Ah, yeah, I see that now.